xbmc/video/dialogs/GUIDialogVideoManager.cpp

```diff
@@ -330,6 +330,15 @@ void CGUIDialogVideoManager::ChooseArt()
   if (!CGUIDialogVideoInfo::ChooseAndManageVideoItemArtwork(m_selectedVideoAsset))
     return;
 
+  m_videoAsset->ClearArt();
+  CVideoThumbLoader thumbLoader;
+  thumbLoader.LoadItem(m_videoAsset.get());
+
+  // notify all windows to update the file item
+  CGUIMessage msg{GUI_MSG_NOTIFY_ALL,        0,           0, GUI_MSG_UPDATE_ITEM,
+                  GUI_MSG_FLAG_FORCE_UPDATE, m_videoAsset};
```

Comment:
This formatting required by clang / Jenkins. There has to be a way to tune this?

Reply:
No idea.

Reply:
In clang-format we trust. But it subjectively flunks some cases. Not hard to fix:

```
// clang-format off
...
// clang-format on`
```

Reply:
yes thanks, as last resort.